docs/admin/addons.rst

```diff
@@ -113,17 +109,12 @@ Component discovery
 Automatically adds or removes project components based on file changes in the
 version control system.
 
-It is triggered on every VCS update, and otherwise similar to the :djadmin:`import_project`
-management command. This way you can track multiple translation
-components within one VCS.
-
-Create one main component least likely to disappear in the future, and others
-will employ :ref:`internal-urls` to it as a VCS configuration, and configure it
-to find all components in it.
+Triggered each time the VCS is updated, and otherwise similar to
+the :djadmin:`import_project` management command. This way you can track
+multiple translation components within one VCS.
 
-The matching is done using regular expressions, where power is a tradeoff for
-complexity in configuration. Some examples for common use cases can be found in
-the addon help section.
+Matching is done using powerful regular expressions, reducing configuration
```

Comment:
configuration complexity of the UI? Why mention its relation to configuration complexity?

Reply:
If you are not sure about something that much that you ask about that, please don’t change it. At least wait for the answer before the change.